internal/pkg/build/docker/docker.go

```diff
@@ -13,23 +13,42 @@ import (
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
 )
 
-// TODO: wrap the `os/exec` stuff in a `command` package to make mocking and testing easier on this package.
+type command struct {
+	cmd *exec.Cmd
+}
+
+func (c command) run() error {
+	return c.cmd.Run()
+}
+
+func (c command) standardInput(input string) {
+	c.cmd.Stdin = strings.NewReader(input)
+}
+
+type runnable interface {
+	run() error
+	standardInput(string)
+}
 
 // Service exists for mockability.
-type Service struct{}
+type Service struct {
+	createCommand func(string, ...string) runnable
```

Comment:
nit: can we give names to the parameters to make it easier to parse since they're the same type